# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pyrexc/distutils-swig-hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Pyrex/Distutils/build_ext.py.orig	2005-08-31 11:53:55.488620976 -0400
+++ ./Pyrex/Distutils/build_ext.py	2005-08-31 11:54:28.548148376 -0400
@@ -32,7 +32,7 @@
       if self.compiler == 'mingw32':
         self.swig_cpp = 1
 
-  def swig_sources (self, sources):
+  def swig_sources (self, sources,extension=None):
     if not self.extensions:
       return
 
